feat: add new icons for opentofu

Moulick opened the pull request:
# Description

Add icons for OpenTofu. Files pulled from https://github.com/opentofu/brand-artifacts/tree/main/symbol-only/transparent/SVG and adjusted to comply with material colors, minor resizing to fit the grid

Fixes: https://github.com/material-extensions/vscode-material-icon-theme/issues/2736

## Contribution Guidelines

- [x] By creating this pull request, I acknowledge that I have read the [Contribution Guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md) for this project
- [x] I have read the [Code Of Conduct](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CODE_OF_CONDUCT.md) and promise to abide by these rules

lucas-labs reviewed (commented):
> Editing the files with inkscape as left some references to it in the svg, do they need to be removed?

They will be removed by `svgo` automatically before releasing, so, don't worry. You can run svgo on your own too, as it's installed in the package.json as a tool in this repo, but although is recommended, it's not 100% required.

---

We usually leave at least 1px (at 16px) of margin, that's to give icons some more space when viewed in the tree view in vscode. Can you fix that (also applies for the borders in the light version)? 😁

<img width="941" height="426" alt="image" src="https://github.com/user-attachments/assets/6b0aa052-fbf9-4f1d-863f-e1bb90bc51f3" />

lucas-labs commented:
> ...
> 
> <img alt="image" width="200" src="https://private-user-images.githubusercontent.com/12949236/486226797-6b0aa052-fbf9-4f1d-863f-e1bb90bc51f3.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcwOTYyMjIsIm5iZiI6MTc1NzA5NTkyMiwicGF0aCI6Ii8xMjk0OTIzNi80ODYyMjY3OTctNmIwYWEwNTItZmJmOS00ZjFkLTg2M2YtZTFiYjkwYmM1MWYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA1VDE4MTIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExM2M1NjRiNjE5YWRiNWQwNDliZTBlOTdhNjdhYTg0MjU0OWY4Yzk4MmU0OGQxMGI4OGI4Njk0ZTEwYzcwYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5VnQyjC3J_vMBVSQxui5zk4aWRa3zeGjw8lTX-zoKoY">

This screenshot comes from [here](https://pixp.pages.dev/material-extensions/vscode-material-icon-theme/pull/3177), btw, which you can also use to check if your icon aligns with the grid. It will be hard to exactly align this one, as it has "diagonals" and there's no "center" pixels in a 16x16 grid, but at least we could try to make the left and right borders align with the grid.

https://pixp.pages.dev/material-extensions/vscode-material-icon-theme/pull/3177

lucas-labs commented:
> > We usually leave at least 1px (at 16px) of margin, that's to give icons some more space when viewed in the tree view in vscode. Can you fix that (also applies for the borders in the light version)? 😁
> 
> I left 1px at 32x32 so making that 2px at 32x32 should be good I suppose? I did try that first but it made the icon look fat :D I'll try leaving 4px left/right and 2px top/bottom

Yes, I also use 32px artboards, and divide everything by 2 to make it a 16px grid. So, 2px = 1px as you said.

Moulick commented:
<img width="1102" height="969" alt="image" src="https://github.com/user-attachments/assets/1bd87a01-9ba5-4e62-83fc-7803870bdf24" />

Moulick commented:
> > Editing the files with inkscape as left some references to it in the svg, do they need to be removed?
> 
> They will be removed by `svgo` automatically before releasing, so, don't worry. You can run svgo on your own too, as it's installed in the package.json as a tool in this repo, but although is recommended, it's not 100% required.

running `bun run svgo` is updating a lot more icons than just opentofu. I can open a separate PR before this PR to cleanup the other ones?

```
	modified:   icons/cursor.svg
 	modified:   icons/cursor_light.svg
	modified:   icons/folder-cursor-open.svg
	modified:   icons/folder-cursor-open_light.svg
	modified:   icons/folder-cursor.svg
	modified:   icons/folder-cursor_light.svg
	modified:   icons/folder-metro-open.svg
	modified:   icons/folder-metro.svg
	modified:   icons/metro.svg
	modified:   icons/onnx.svg
	modified:   icons/opentofu.svg
	modified:   icons/opentofu_light.svg
	modified:   icons/quarto.svg
	modified:   icons/verse.svg
	modified:   icons/verse_light.svg
```

lucas-labs commented:
> running `bun run svgo` is updating a lot more icons than just opentofu. I can open a separate PR before this PR to cleanup the other ones so as to keep git clean.

Oh yeah, that's because all those svgs have not been released just yet. They will be automatically "fixed" by svgo as part of the automatic release workflow. If you want to pass your icons through svgo yourself, you will have to revert the others before committing, to keep your PR clean. But don't worry, as I said, there's a github action setted up that optimizes all of our icons before releasing.

Moulick commented:
Thanks @lucas-labs and @PKief.

Moulick commented:
When are releases usually done?
